RFC: TOC can stand in for WG leads (#676)
cloudfoundry/community: toc/rfc/rfc-draft-toc-stand-in-for-wg-lead.md

# Meta
[meta]: #meta
- Name: TOC Can Stand in for Working Group Leads
- Start Date: 2023-08-28
- Author(s): Amelia Downs
- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
- RFC Pull Request: https://github.com/cloudfoundry/community/pull/676

## Summary

Working Group Leads are in charge of approving and merging PRs for their

> Working Group. When the TOC decideds it is necessary, the TOC MAY make decisions in their stead.

Author: ```suggestion
Working Group. When the TOC decideds it is necessary, the TOC MAY make decisions in their place.
```

## Problem

Sometimes Working Group Leads go on vacation or get sick and are not available

> to review PRs. This can cause progress on their teams to grind to halt until
> the Working Group lead returns.

Author: ```suggestion
to take actions for their working group. This can cause work on their working groups to be blocked until
the Working Group Lead returns.
```

## Proposal

> When the TOC decideds it is necessary, the TOC MAY make decisions in their stead. This

Author: ```suggestion
When the TOC decides it is necessary, the TOC MAY make decisions in their stead. This
```
